docs/getting-started/operator-quick-start.md

```diff
@@ -0,0 +1,429 @@
+  creationTimestamp: "2024-12-18T22:22:52Z"
+  finalizers:
+  - bpfman.io.operator/finalizer
+  generation: 2
+  labels:
+    app.kubernetes.io/name: tcnsprogram
+  name: tc-containers
+  namespace: acme
+  resourceVersion: "7993"
+  uid: 49291f28-49dc-4486-9119-af7c31569de3
+spec:
+  bpffunctionname: pass
+  bytecode:
+    image:
+      imagepullpolicy: IfNotPresent
+      url: quay.io/bpfman-bytecode/tc_pass:latest
+  containers:
+    containernames:
+    - nginx
+    pods:
+      matchLabels:
+        app: nginx
+  direction: ingress
+  globaldata:
+    GLOBAL_u8: AQ==
+    GLOBAL_u32: DQwLCg==
+  interfaceselector:
+    interfaces:
+    - eth0
+  mapownerselector: {}
+  nodeselector: {}
+  priority: 0
+  proceedon:
+  - pipe
+  - dispatcher_return
+status:
+  conditions:
+  - lastTransitionTime: "2024-12-18T22:23:11Z"
+    message: bpfProgramReconciliation Succeeded on all nodes
+    reason: ReconcileSuccess
+    status: "True"
+    type: ReconcileSuccess
+```
+
+To view each attachment point on each node, use the `bpfNsProgram` object:
+
+```bash
+$ kubectl get bpfnsprograms
+NAME                     TYPE   STATUS         AGE
+tc-containers-6494dbed   tc     bpfmanLoaded   12m
+tc-containers-7dcde5ab   tc     bpfmanLoaded   12m
+
+
+$ kubectl get bpfnsprograms tc-containers-6494dbed -o yaml
+apiVersion: bpfman.io/v1alpha1
+kind: BpfNsProgram
+metadata:
+  annotations:
+    bpfman.io.tcnsprogramcontroller/containerpid: "3256"
+    bpfman.io.tcnsprogramcontroller/interface: eth0
+    bpfman.io/ProgramId: "26575"
+    bpfman.io/bpfProgramAttachPoint: eth0-ingress-nginx-deployment-57d84f57dc-lgc6f-nginx
+  creationTimestamp: "2024-12-18T22:23:08Z"
+  finalizers:
+  - bpfman.io.tcnsprogramcontroller/finalizer
+  generation: 1
+  labels:
+    bpfman.io/appProgramId: ""
+    bpfman.io/ownedByProgram: tc-containers
+    kubernetes.io/hostname: bpfman-deployment-control-plane
+  name: tc-containers-6494dbed
+  namespace: acme
+  ownerReferences:
+  - apiVersion: bpfman.io/v1alpha1
+    blockOwnerDeletion: true
+    controller: true
+    kind: TcNsProgram
+    name: tc-containers
+    uid: 49291f28-49dc-4486-9119-af7c31569de3
+  resourceVersion: "7992"
+  uid: c913eea4-71e0-4d5d-b664-078abac36c40
+spec:
+  type: tc
+status:
+  conditions:
+  - lastTransitionTime: "2024-12-18T22:23:11Z"
+    message: Successfully loaded bpfProgram
+    reason: bpfmanLoaded
+    status: "True"
+    type: Loaded
+```
+
+## API Types Overview
+
+Refer to  [api-spec.md](../developer-guide/api-spec.md) for a more detailed description of all the bpfman Kubernetes API types.
+
+### Cluster Scoped Versus Namespaced Scoped CRDs
```

Comment:
It seems like this info would be useful before we go through the cluster and namespace examples above.

Reply:
I moved the entire `## API Types Overview` section up, after bpfman-operator and bpfman-daemon have been deployed and verified, but before any of the samples have been run.